Assets/Scripts/Character/PlayerInput.cs

```diff
@@ -27,8 +27,8 @@ public class PlayerInput : BaseInput {
     Controls.Player.Move.canceled += ctx => { activeMovementInput = false; movement.Move(Vector2.zero); movement.ToggleSprint(false);};
     Controls.Player.Sprint.performed += ctx => movement.ToggleSprint(true);
     Controls.Player.Jump.performed += ctx => OnJump();
-    Controls.Player.LookAtWatch.performed += ctx => movement.LookAtWatch();
-    Controls.Player.LookAtWatch.canceled += ctx => movement.PutWatchAway();
+    Controls.Player.LookAtWatch.performed += ctx => { if ((!dialogueManager) || (!dialogueManager.isDialoging)) movement.LookAtWatch();};
```

Comment:
The dialogue manager should just disable the `Player` controls and enable the `UI` controls when it's dialoging. See `UIEventListener` for an example of this.

Reply:
The only issue with this, is that the way I set it up was that I wanted to leave use of the manager flexible. So we may have managers for different sets of dialogue (UI, conversations, whatever else). I touched on this already in the DialogueTrigger.cs comment. 

So if we do have multiple managers later, disabling Player controls each time might cause issues. Lmk if you think we should make the dialogue manager a singleton and then I'll go ahead and make that change in the next PR